tools/lib/src/publish_plugin_command.dart

```diff
@@ -62,7 +55,7 @@ class PublishPluginCommand extends PackageLoopingCommand {
       help:
           'Skips the real `pub publish` command and assumes the command is successful.\n'
           'This does not run `pub publish --dry-run`.\n'
-          'If you want to run the command with `pub publish --dry-run`, use `pub-publish-flags=--dry-run`',
+          'If you want to run the command with `pub publish --dry-run`, use `--pub-publish-flags=--dry-run`',
       defaultsTo: false,
       negatable: true,
```

Comment:
It's out of the scope of this PR but consider applying https://github.com/flutter/plugins/commit/cda37d5640f9fe1d4b1bac9a6e610d5ca4e867ee#diff-c726ae6071da1e3893cc96959dccc1b9e58412242151a277cde199c33a271d26 later.

Reply:
Okay, thanks for the info!